bin/BuildPackages.sh

```diff
@@ -86,6 +105,28 @@ then
   GAPDIR="$(pwd)"
   popd
 fi
+
+# check if coloring should be turned off
+if [[ "$#" -ge 1 ]]
+then
+  case "$1" in
+    --no-color|--no-colors|--no-colour|--no-colours|\
+    --nocolor|--nocolors|--nocolour|--nocolours)
+      COLOR=0
+      shift
+    ;;
+  esac
+fi
```

Comment:
Just to be clear, is this what you want?

- have the build_one_package function which will be called for the packages and make that log to log/$PKG.log or wherever. 

- do not care about logging anything else, e.g. if wrong GAPDIR directory is added. 

- parse the arguments immediately and do the necessary checks, and then call build_one_package on the appropriate packages.

Happy to do it, just do not want to do it twice....